mmcls/models/backbones/resnet.py

```diff
@@ -518,6 +530,12 @@ def __init__(self,
         self.res_layers = []
         _in_channels = stem_channels
         _out_channels = base_channels * self.expansion
+
+        # stochastic depth decay rule
+        net_num_blocks = sum(stage_blocks)
+        dpr = np.linspace(0, drop_path_rate, net_num_blocks)
+        block_id = 0
```

Comment:
Maybe directly use a list instead of numpy array. refers to the SwinTransformer. 
```suggestion
        # stochastic depth decay rule
        total_depth = sum(stage_blocks)
        dpr = [
            x.item() for x in torch.linspace(0, drop_path_rate, total_depth)
        ]
```